service/src/main/java/uk/gov/hmcts/reform/fpl/enums/MandatoryDocuments.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.fpl.enums;
```

Comment:
Minor thing, but if this is "user interface thing" as the second value suggest, not sure if it should belong to all enums package. 

Somethign to keep in mind, but if you are working on service level you shouldn't see anything related to UI.

Reply:
Maybe move to Utils?

Reply:
Not sure if utils is bet was thinking something UI related but let's leave it as it is 👍